hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestMaster.java

```diff
@@ -100,6 +104,25 @@ public static void afterAllTests() throws Exception {
     TEST_UTIL.shutdownMiniCluster();
   }
 
+  @Test
+  public void testRegionServerStartup() throws Exception {
```

Comment:
nit: Add some detail about what this test does? (preferably with a reference to the jira ID).